newIDE/app/src/Utils/UpdateDiscordRichPresence.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import optionalRequire from './OptionalRequire';
+const electron = optionalRequire('electron');
+const ipc = electron ? electron.ipcRenderer : null;
+
+/*::
```

Comment:
In the IDE, use direct Flow types (without comments) if the file is not used outside of the IDE.